src/widgets/numeric-menu/__tests__/numeric-menu-test.ts

```diff
@@ -55,23 +66,25 @@ describe('numericMenu()', () => {
     });
 
     helper = algoliasearchHelper(
-      {},
+      createSearchClient(),
       '',
       widget.getWidgetSearchParameters(new SearchParameters(), { uiState: {} })
     );
 
     jest.spyOn(helper, 'search');
 
     state = helper.state;
-    results = new SearchResults(helper.state, [{ nbHits: 0 }]);
+    results = new SearchResults(helper.state, [
+      createSingleSearchResponse({ nbHits: 0 }),
+    ]);
 
     createURL = () => '#';
-    widget.init({ helper, instantSearchInstance: {} });
+    widget.init!({ helper, instantSearchInstance: {} });
   });
 
   it('calls twice render(<RefinementList props />, container)', () => {
-    widget.render({ state, results, createURL });
-    widget.render({ state, results, createURL });
+    widget.render!({ state, results, createURL });
```

Comment:
Any reasons the `createRenderOptions` and `createInitOptions` are not used below?

Reply:
@francoischalifour Nice catch.

Reply:
Fixed. Note that the snapshots got updated as we use a templateConfig as object instead of undefined.